changelog.d/20240229_151232_dawoud.sheraz_py312.md

```diff
@@ -0,0 +1 @@
+- [Feature] Add Python 3.12 CI and `make docs` compatibility (by @DawoudSheraz)
```

Comment:
CI-only changes don't warrant a changelog. 

Basically, to decide whether a change needs a changelog entry, we need to place ourselves in the position of the typical Tutor user: platform administrator or plugin developer. Whenever they upgrade, we expect that they have a look at the changelog entries. We don't want them to worry unnecessarily about changes that won't impact them. But we do want to highlight the features that they might want to use and the breaking changes.

Reply:
Ok.